tests/helpers/errors.py

```diff
@@ -4,31 +4,23 @@ def get_error_test_cases(errors):
 
 ERRORS = {
     'invalid-parameters': {
-        "parameter": {
-            "example": [
-                "The example field is required"
-            ]
-        },
         "type": "https://www.rev.ai/api/v1/errors/invalid-parameters",
-        "title": "Your request parameters didn't validate",
```

Comment:
this breaks regex on the test verification so easiest to just cut it out. I'm guessing this is why it was left out